numpy/f2py/tests/test_crackfortran.py

```diff
@@ -89,6 +89,41 @@ def test_defaultPublic(self, tmp_path):
         assert 'public' in mod['vars']['seta']['attrspec']
 
 
+class TestModuleProcedure():
+    def test_defaultPrivate(self, tmp_path):
+        f_path = tmp_path / "mod.f90"
+        with f_path.open('w') as ff:
+            ff.write(textwrap.dedent("""\
+            module foo
+              interface operator(.item.)
+                module procedure item_char, item_dict, item_dbl
+              end interface
+              interface operator(==)
+                module procedure dicts_are_equal
+              end interface
+              interface assignment(=)
+                module procedure get_value, get_integer, get_real
+              end interface
+            contains
+
+            end module foo
```

Comment:
Could this be replaced with a full Fortran snippet? currently without a function or subroutine in contains this is incorrect.

Reply:
That's a good remark. I've updated the PR with a full Fortran snippet that compiles with GFortran and `-Wall -Werror`. Thanks for for pointing it out.